Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/User/AdditionalUserFieldsWindow.tsx

```diff
@@ -89,6 +89,8 @@ function AdditionalField(props: IProps) {
 	const [errorFields, setErrorFields] = React.useState<string[]>([]);
 	const [fieldErrors, setFieldErrors] = React.useState<string[]>([]);
 
+	const [editNew, setEditNew] = React.useState<Application.Types.NewEdit>('New');
+
```

Comment:
is this replacing `showEdit`? we may want to do that and use `NewEdit` variables everywhere instead of a mic of `boolean` and `NewEdit`